src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -248,9 +246,13 @@ private Dispatcher Dispatcher
 
         private static MetadataFileReferenceResolver CreateFileResolver(ImmutableArray<string> referencePaths, string baseDirectory)
         {
+            var userProfilePath = Environment.GetEnvironmentVariable("userprofile");
+            var packagesDirectory = (userProfilePath == null) ?
```

Comment:
There's also a NUGET_PACKAGES environment variable that is respected. You might want to see if there's a flag you can pass to NuGet and restore explicitly rather than hoping your logic is the same as their logic.